server/utils/bedspaceSearchResultUtils.test.ts

```diff
@@ -50,4 +51,12 @@ describe('BedspaceSearchResultUtils', () => {
       ])
     })
   })
+
+  describe('premisesNotes', () => {
+    it('returns escaped premises notes', () => {
```

Comment:
Could you add a test to show that the notes are actually formatted?

Example: If I pass a string that contains line breaks I would expect it to return the notes with <br> or <p> tags